--- php-simplepie-1.2/simplepie.inc~	2007-07-23 10:35:34.000000000 +0300
+++ php-simplepie-1.2/simplepie.inc	2009-12-12 06:35:10.000000000 +0200
@@ -51,13 +51,12 @@
 /**
  * SimplePie Version
  */
-define('SIMPLEPIE_VERSION', '1.2.1-dev');
+define('SIMPLEPIE_VERSION', '1.2.1');
 
 /**
  * SimplePie Build
- * @todo Hardcode for release (there's no need to have to call SimplePie_Misc::parse_date() only every load of simplepie.inc)
  */
-define('SIMPLEPIE_BUILD', gmdate('YmdHis', SimplePie_Misc::parse_date(substr('$Date$', 7, 25)) ? SimplePie_Misc::parse_date(substr('$Date$', 7, 25)) : filemtime(__FILE__)));
+define('SIMPLEPIE_BUILD', 20091211063510);
 
 /**
  * SimplePie Website URL
